system/lib/pthread/pthread_create.c

```diff
@@ -244,7 +244,7 @@ void _emscripten_thread_exit(void* result) {
   }
 }
 
-// Mark as `no_sanitize("address"` since emscripten_pthread_exit destroys
+// Mark as `no_sanitize("address")` since emscripten_pthread_exit destroys
```

Comment:
Looks like all the changes to this file are just cleanups.. can you split this out?

Reply:
OK, reverted those with commit 72da048d62269fb9b20b2fc23e011594cfeeb81a. I'll make a new PR with these changes soon (perhaps with `[skip ci]` in the commit message, since these are all minor changes?).